support-frontend/assets/pages/digital-subscription-checkout/thankYouGift.jsx

```diff
@@ -246,7 +246,7 @@ function ThankYouGift(props: PropTypes) {
             <ul>
               <li css={detailsWithIconList}>
                 <div css={iconContainer}><img src={gift} alt="" /></div>
-                <div css={giftStep}>{props.giftRecipient} will receive an email on the date you&apos;ve chosen with
+                <div css={giftStep}>{props.giftRecipient ? props.giftRecipient : 'Your recipient'} will receive an email on the date you&apos;ve chosen with
```

Comment:
Just a small thing, but we could use an or statement in these instances rather than a ternary

```js
{props.giftRecipient || 'Your recipient'}
```